recipe/bld.bat

```diff
@@ -1,18 +1,8 @@
-set LIB=%LIBRARY_LIB%;%LIB%
-set LIBPATH=%LIBRARY_LIB%;%LIBPATH%
-set INCLUDE=%LIBRARY_INC%;%INCLUDE%;%RECIPE_DIR%
 
-set GEOS_LIBRARY_PATH=%LIBRARY_BIN%\geos_c.dll
+set GEOS_LIBRARY_PATH=%LIBRARY_LIB%
 
-del /f shapely\speedups\_speedups.c
-del /f shapely\vectorized\_vectorized.c
+"%PYTHON%" -m cython shapely\speedups\_speedups.pyx
+"%PYTHON%" -m cython shapely\vectorized\_vectorized.pyx
```

Comment:
Those two lines might not actually be needed? (I would expect that the `setup_py build_ext` will do that as well)

Reply:
attempting that now.